src/main/java/com/endava/internship/collections/Node.java

```diff
@@ -0,0 +1,18 @@
+package com.endava.internship.collections;
+
+public class Node {
```

Comment:
Overall correct solution, however this class is only used in StudentMap so we could realistically have this as an inner static class in StudentMap.